service/src/integrationTest/kotlin/fi/espoo/evaka/occupancy/RealtimeOccupancyTest.kt

```diff
@@ -236,6 +236,30 @@ class RealtimeOccupancyTest : FullApplicationTest(resetDbBeforeEach = true) {
             ?: error("data point missing")
     }
 
+    @Test
+    fun childWithoutPlacementGetsCapacityFactor1() {
+        db.transaction { tx ->
+            FixtureBuilder(tx, date)
+                    .addChild()
+                    .withAge(2, 10)
+                    .saveAnd {
+                        // addPlacement().toUnit(testDaycare.id).save()
+                        addAttendance()
+                                .inUnit(testDaycare.id)
+                                .arriving(LocalTime.of(7, 45))
+                                .departing(LocalTime.of(16, 30))
+                                .save()
+                    }
+        }
+
+        val result = getRealtimeOccupancy()
+        val occupancies = result.occupancySeries
+        assertEquals(2, occupancies.size)
+
+        occupancies.get(0)
+                ?.also { assertEquals(1.0, it.childCapacity) }
```

Comment:
Wouldn't `occupancies.get(0)?.also { ... }` possibly let a bug through? This would be better:
```
assertEquals(1.0, occupancies[0].childCapacity)
```